app/os_windows.go

```diff
@@ -749,7 +749,13 @@ func loadCursor(cursor pointer.Cursor) (syscall.Handle, error) {
 	}
 }
 
-func (w *window) ShowTextInput(show bool) {}
+func (w *window) ShowTextInput(show bool) {
+	if show {
+		windows.SetFocus(w.hwnd)
+	} else {
+		windows.SetFocus(0)
```

Comment:
Why force focus away from the window when `show` is false? On the other platforms dismissing software text input doesn't imply a focus loss, does it?